test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
Yeah, it's just what I'm used to looking at I think. The advantage of doing
output_rate or whatever you'd call it is it wouldn't be inverted, and
depending on what the api client wanted to do, could lead to more accurate
math.

Consider that you've grouped the orders into 4 buckets. To determine how
much X you'd get for n Y (execution prediction) you'd need to do two
different kinds of math anyway. One using offers that are 'selling x' and
the other for 'buying y'. There's no nice and simple 'price' to apply.

I think output_rate is at least clearer than quality fwiw. I think most ppl
could deduce the meaning, once they worked out that an offer has
inputs/outputs.